internal/app/splinter/api/api.go

```diff
@@ -0,0 +1,263 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/mp-hl-2021/splinter/internal/app/splinter/model"
+	"log"
+	"net/http"
+	"reflect"
+	"strings"
+)
+
+type Api struct {
+	model.UserInterface
+}
+
+func NewApi(x model.UserInterface) Api {
+	return Api{x}
+}
+
+type Context struct {
```

Comment:
I'd rather suggest using some other name here to avoid future conflicts with the standard package https://golang.org/pkg/context/